<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="" tests="12" assertions="24" errors="0" warnings="0" failures="0" skipped="0" time="0.005058">
    <testsuite name="Test suite" tests="12" assertions="24" errors="0" warnings="0" failures="0" skipped="0" time="0.005058">
      <testsuite name="Application\Tests\Controller\Common\ControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Common/ControllerTest.php" tests="3" assertions="6" errors="0" warnings="0" failures="0" skipped="0" time="0.002999">
        <testcase name="testIHaveCorrectResponseWhenIUseGetResponseJson" class="Application\Tests\Controller\Common\ControllerTest" classname="Application.Tests.Controller.Common.ControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Common/ControllerTest.php" line="32" assertions="2" time="0.002791"/>
        <testcase name="testIHaveCorrectResponseWhenIUseGetResponseJsonSuccess" class="Application\Tests\Controller\Common\ControllerTest" classname="Application.Tests.Controller.Common.ControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Common/ControllerTest.php" line="46" assertions="2" time="0.000124"/>
        <testcase name="testIHaveCorrectResponseWhenIUseGetResponseJsonError" class="Application\Tests\Controller\Common\ControllerTest" classname="Application.Tests.Controller.Common.ControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Common/ControllerTest.php" line="60" assertions="2" time="0.000085"/>
      </testsuite>
      <testsuite name="Application\Tests\Controller\Error\ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" tests="7" assertions="14" errors="0" warnings="0" failures="0" skipped="0" time="0.001787">
        <testsuite name="Application\Tests\Controller\Error\ErrorControllerTest::testIGetCompleteErrorResponseWhenICallErrorController" tests="7" assertions="14" errors="0" warnings="0" failures="0" skipped="0" time="0.001787">
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpBadRequestException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.001003"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpUnauthorizedException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000155"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpForbiddenException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000116"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpNotFoundException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000111"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpMethodNotAllowedException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000110"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;HttpServiceUnavailableException&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000109"/>
          <testcase name="testIGetCompleteErrorResponseWhenICallErrorController with data set &quot;default exception&quot;" class="Application\Tests\Controller\Error\ErrorControllerTest" classname="Application.Tests.Controller.Error.ErrorControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Error/ErrorControllerTest.php" line="43" assertions="2" time="0.000182"/>
        </testsuite>
      </testsuite>
      <testsuite name="Application\Tests\Controller\Health\HealthControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Health/HealthControllerTest.php" tests="2" assertions="4" errors="0" warnings="0" failures="0" skipped="0" time="0.000272">
        <testcase name="testEndpointLivenessReturnStatusOkWhenItIsCalled" class="Application\Tests\Controller\Health\HealthControllerTest" classname="Application.Tests.Controller.Health.HealthControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Health/HealthControllerTest.php" line="34" assertions="2" time="0.000192"/>
        <testcase name="testEndpointReadinessReturnStatusOkWhenItIsCalledWithOtherServicesReady" class="Application\Tests\Controller\Health\HealthControllerTest" classname="Application.Tests.Controller.Health.HealthControllerTest" file="/home/ymar/projects/d4c-backend-test-master/tests/Controller/Health/HealthControllerTest.php" line="49" assertions="2" time="0.000080"/>
      </testsuite>
    </testsuite>
  </testsuite>
</testsuites>
